code/__DEFINES/sound.dm

```diff
@@ -1,15 +1,22 @@
-//max channel is 1024. Only go lower from here, because byond tends to pick the first available channel to play sounds on
-#define CHANNEL_LOBBYMUSIC 1024
-#define CHANNEL_ADMIN 1023
-#define CHANNEL_VOX 1022
-#define CHANNEL_JUKEBOX 1021
-#define CHANNEL_HEARTBEAT 1020 //sound channel for heartbeats
-#define CHANNEL_BOSS_MUSIC 1019
-#define CHANNEL_AMBIENCE 1018
-#define CHANNEL_BUZZ 1017
-#define CHANNEL_TRAITOR 1016
-#define CHANNEL_CHARGED_SPELL 1015
-#define CHANNEL_ELEVATOR 1014
+// DO NOT ADD ANYMORE SOUND CHANNELS. REFACTOR THE SOUND SUBSYSTEM TO ALLOW NAMED CHANNEL RESERVATIONS INSTEAD
```

Comment:
Refactor what? You can already reserve sound channels with an owner datum, or no datum at all.